### DISCUSSION TITLE
[add photos in album] via 'folder' option and by Search

### DISCUSSION THREAD

**Discussion by balvant813** [immich-app/immich#13680] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Need a feature to add photos by choosing Immich 'folder' option.  This be very useful  as new photos are generally added in a separate folder  which added into Immich via 'External Link'.  The new photos likely to have various dates.  if  one is adding photos from few years back, adding photos in album via timeline is hard as it in compressed view.

Alternatively,  a search and add option will also be a great feature.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4279
